src/main/java/pl/simpleascoding/tutoringplatform/dto/ChangeUserPasswordDTO.java

```diff
@@ -1,14 +1,12 @@
 package pl.simpleascoding.tutoringplatform.dto;
 
-import org.hibernate.annotations.Immutable;
-
 /**
  * Record used to transfer data (passwords) between the service and the controller
  *
- * @param oldPassword   The user's existing password
- * @param newPassword   New user password
- * @param confirmationPassword   Confirmation of the new user password
+ * @param oldPassword          The user's existing password
+ * @param newPassword          New user password
+ * @param confirmationPassword Confirmation of the new user password
  */
-@Immutable
+
```

Comment:
Please remove empty line between documentation comment and code.

Reply:
Done